scripts/virtualenv.mk

```diff
@@ -0,0 +1,41 @@
+
```

Comment:
Would it be applicable to use wake's python virtual env support from api-languages-sifive/python.wake

Reply:
Make is first-class in freedom-e-sdk because it's what our customers have and want.

Does api-languages-sifive handle the "cache all prebuilt wheels for common platforms" flow we need here to make sure that customers don't have to hit PyPI or need a C compiler to create the virtualenv?